### ISSUE TITLE
Runtime Issue. Publishing changes from within view updates is not allowed. Xcode 14.0 beta 5

### ISSUE THREAD

**Issue by artbobrov** [gonzalezreal/AttributedText#23]
SwiftUI Runtime Issue: `Publishing changes from within view updates is not allowed, this will cause undefined behavior.`

Xcode version: `14.0 beta 5`

<img width="1074" alt="Screen Shot 2022-08-20 at 1 54 27 PM" src="https://user-images.githubusercontent.com/24722656/185742744-b3249fdc-31bc-4fb8-abdf-b466c8620476.png">

Can be reproduced by using this code:

```swift
import SwiftUI
import AttributedText


struct ContentView: View {
    var body: some View {
        AttributedText {
            NSAttributedString(string: "Hello world")
        }
    }
}
```

P.S. This issue also effects `MarkdownUI`

**Comment by kaylanx** [comment]
I was so happy when I found this library, as it solved my problem height problem.  
However now it seems to be showing this error :(

**Comment by waterskier2007** [comment]
@gonzalezreal with the change merged and released in 1.0.1, should this issue be closed?
